app/config/FrontendAppConfig.scala

```diff
@@ -125,9 +121,6 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
     "cymraeg" -> Lang("cy")
   )
 
-  def routeToSwitchLanguage: String => Call =
-    (lang: String) => routes.LanguageSwitchController.switchToLanguage(lang)
-
```

Comment:
Let's leave this one in for potential future changes.

Reply:
Done